tools/workflow/cleaner/autoscaling/clean.go

```diff
@@ -28,9 +28,9 @@ import (
 
 const Type = "autoscaling"
 
-var logger = log.New(os.Stdout, fmt.Sprintf("[%s] ", Type), log.Ldate)
+var logger = log.New(os.Stdout, fmt.Sprintf("[%s] ", Type), log.LstdFlags)
 
-func Clean(sess *session.Session, keepDuration time.Duration) error {
+func Clean(sess *session.Session, expirationDate time.Time) error {
```

Comment:
I would like to request a comment be added to explain the usage of expiration date and how it is utilized. Readability is a little hard since it is being compared to resource creation date.